src/common/types.mo

```diff
@@ -18,6 +18,8 @@ module {
     decideid_canister_id : Principal;
   };
 
+  public type Timestamp = Int;
```

Comment:
it must be Nat as Time cant be negative

Reply:
em. I cannot use Nat. The lib we are using define the time as Int:
`.vessel/base/494824a2787aee24ab4a5888aa519deb05ecfd60/src/Time.mo` 
```
import Prim "mo:⛔";
module {

  /// System time is represent as nanoseconds since 1970-01-01.
  public type Time = Int;

  /// Current system time given as nanoseconds since 1970-01-01. The system guarantees that:
  ///
  /// * the time, as observed by the canister smart contract, is monotonically increasing, even across canister upgrades.
  /// * within an invocation of one entry point, the time is constant.
  ///
  /// The system times of different canisters are unrelated, and calls from one canister to another may appear to travel "backwards in time"
  ///
  /// Note: While an implementation will likely try to keep the system time close to the real time, this is not formally guaranteed.
  public let now : () -> Time =
    func () : Int = Prim.nat64ToNat(Prim.time());
```

Reply:
BTW, `  public type Timestamp = Int;` is commonly used in our repo.